docs/developer-docs/latest/developer-resources/database-apis-reference/entity-service/filter.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Entity Service API - Filtering - Strapi Developer Documentation
+description: Filter, order and paginate results with Entity Service API
+sidebarDepth: 3
+---
+
+<!-- TODO: update SEO -->
+
+# Entity Service API: Filtering
+
+The [Entity Service API](/developer-docs/latest/developer-resources/database-apis-reference/entity-service-api.md) offers the ability to filter results found with its [findMany()](/developer-docs/latest/developer-resources/database-apis-reference/entity-service/crud.html#findmany) method.
+
+Results are filtered with the `filters` parameter that accepts [logical operators](#logical-operators) and [attribute operators](#attribute-operators). Every operator should be prefixed with `$`.
+
+## Logical operators
```

Comment:
We could explain $and $or and $not are nestable inside of other $and $or and $not